src/humanize/time.py

```diff
@@ -173,6 +173,9 @@ def naturaldelta(
         minimum_unit (str): The lowest unit that can be used.
         when (datetime.datetime): Point in time relative to which _value_ is
             interpreted.  Defaults to the current time in the local timezone.
+            .. deprecated:: 3.13
```

Comment:
The next release will be 3.14:

```suggestion
            .. deprecated:: 3.14
```